src/AppInstallerCLICore/Workflows/ShellExecuteInstallerHandler.cpp

```diff
@@ -538,4 +538,36 @@ namespace AppInstaller::CLI::Workflow
             context.Add<Execution::Data::OperationReturnCode>(enableFeatureResult.value());
         }
     }
-}
\ No newline at end of file
+
+    void ShellExecuteExtractArchive::operator()(Execution::Context& context) const
+    {
+        auto tarExecPath = AppInstaller::Filesystem::GetExpandedPath("%windir%\\system32\\tar.exe");
+
+        std::string args = "-xf " + m_archivePath.u8string() + " -C " + m_destPath.u8string();

```

Comment:
Did you try paths with spaces (i.e. do we need to quote the path)?

tar.exe -xf c:\t e s t.zip -C C:\d e s t

Reply:
added single quotes to avoid issues with space in path